packages/synapse-react-client/src/lib/containers/AccessRequirementListV2/AccessRequirementListV2.tsx

```diff
@@ -0,0 +1,347 @@
+        staleTime: Infinity,
+      },
+    )
+
+  const requestDataStepCallback = (props: RequestDataStepCallbackArgs) => {
+    const {
+      managedACTAccessRequirement,
+      step,
+      researchProjectId,
+      dataAccessRequest: newDataAccessRequest,
+    } = props
+    if (managedACTAccessRequirement) {
+      // required for step 1, 2 form
+      setManagedACTAccessRequirement(managedACTAccessRequirement)
+    }
+    if (researchProjectId) {
+      setResearchProjectId(researchProjectId)
+    }
+    if (newDataAccessRequest) {
+      setDataAccessRequest(newDataAccessRequest)
+    }
+    setRequestDataStep(step)
+  }
+
+  const anyARsRequireTwoFactorAuth = accessRequirements?.some(
+    accessRequirement =>
+      accessRequirement.concreteType ===
+        'org.sagebionetworks.repo.model.ManagedACTAccessRequirement' &&
+      accessRequirement.isTwoFaRequired,
+  )
+
+  const anyARsRequireCertification = accessRequirements?.some(
+    accessRequirement =>
+      (accessRequirement.concreteType ===
+        'org.sagebionetworks.repo.model.ManagedACTAccessRequirement' ||
+        accessRequirement.concreteType ===
+          'org.sagebionetworks.repo.model.SelfSignAccessRequirement') &&
+      accessRequirement.isCertifiedUserRequired,
+  )
+
+  const anyARsRequireProfileValidation = accessRequirements?.some(
+    accessRequirement =>
+      (accessRequirement.concreteType ===
+        'org.sagebionetworks.repo.model.ManagedACTAccessRequirement' ||
+        accessRequirement.concreteType ===
+          'org.sagebionetworks.repo.model.SelfSignAccessRequirement') &&
+      accessRequirement.isValidatedProfileRequired,
+  )
+
+  const content = (
+    <>
+      <DialogTitle>
+        <Stack direction="row" alignItems={'center'} gap={'5px'}>
+          Data Access Request
+          <Box sx={{ flexGrow: 1 }} />
+          <IconButton onClick={onHide}>
+            <IconSvg icon={'close'} wrap={false} sx={{ color: 'grey.700' }} />
+          </IconButton>
+        </Stack>
+      </DialogTitle>
+
+      <DialogContent>
+        <DialogSubsectionHeader sx={{ mt: 0 }}>
+          Access For:
+        </DialogSubsectionHeader>
+        <Typography variant={'body1'} component={'span'}>
+          <IconSvg icon="file" sx={{ width: '30px' }} />{' '}
+          {numberOfFilesAffected ? (
+            `${numberOfFilesAffected} File(s)`
+          ) : (
+            <Link
+              href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!Synapse:${entityId}`}
+            >
+              {entityInformation[0]?.name}
+            </Link>
+          )}
+        </Typography>
+        <DialogSubsectionHeader>What do I need to do?</DialogSubsectionHeader>
+        <AuthenticatedRequirement />
+        {anyARsRequireCertification && <CertificationRequirement />}
+        {anyARsRequireProfileValidation && <ValidationRequirement />}
+        {anyARsRequireTwoFactorAuth && <TwoFactorAuthEnabledRequirement />}
```

Comment:
This logical change ends up incurring a slight change to the ordering; we'll always show these requirements first, regardless of if they are met or unmet. This didn't seem to be a problem with ACT/design but I plan to ask again to confirm.